minegauler/shared/utils.py

```diff
@@ -112,10 +112,7 @@ def __str__(self, mapping=None, cell_size=None):
             The size to display a grid cell as. Defaults to the maximum size of
             the representation of all the objects contained in the grid.
         """
-        # @@@LG Some attention please :)
```

Comment:
Don't just remove my comments that say something should be done! I would suggest just ignoring this comment (revert the removal), I may wish to come back to sort this out.